lib/RateLimiterRedis.js

```diff
@@ -105,7 +106,7 @@ class RateLimiterRedis extends RateLimiterStoreAbstract {
     if (secDuration > 0) {
       if(!this.useRedisPackage && !this.useRedis3AndLowerPackage){
         return this.client.rlflxIncr(
-          [rlKey].concat([String(points), String(secDuration)]));
```

Comment:
as I'm passing points as a new arg and not as a new key, we should be good

Reply:
it looks good to me.